datasette/views/table.py

```diff
@@ -543,29 +545,38 @@ async def data(
         sortable_columns = await self.sortable_columns_for_table(
             database, table, use_rowid
         )
-
-        # Allow for custom sort order
-        sort = special_args.get("_sort")
-        sort_desc = special_args.get("_sort_desc")
-
+        #
+        # getting sorting conditions from url and saving them in an list
+        #
+        sort = []
+        sort_desc = []
+
+        if "_sort" in args:
+            for item in args.getlist("_sort"):
+                sort.append(item)
+        if "_sort_desc" in args:
+            for item in args.getlist("_sort_desc"):
+                sort_desc.append(item)
         if not sort and not sort_desc:
-            sort = table_metadata.get("sort")
-            sort_desc = table_metadata.get("sort_desc")
-
-        if sort and sort_desc:
-            raise DatasetteError("Cannot use _sort and _sort_desc at the same time")
-
+            if table_metadata.get("sort"):
+                sort.append(table_metadata.get("sort"))
+            if table_metadata.get("sort_desc"):
+                sort_desc.append(table_metadata.get("sort_desc"))
         if sort:
-            if sort not in sortable_columns:
-                raise DatasetteError(f"Cannot sort table by {sort}")
-
-            order_by = escape_sqlite(sort)
+            for column in sort:
+                if column not in sortable_columns:
+                    raise DatasetteError(f"Cannot sort table by {column}")
 
+            order_by = convert_list_to_sql(sort)
         if sort_desc:
-            if sort_desc not in sortable_columns:
-                raise DatasetteError(f"Cannot sort table by {sort_desc}")
-
-            order_by = f"{escape_sqlite(sort_desc)} desc"
+            for column in sort_desc:
+                if column not in sortable_columns:
+                    raise DatasetteError(f"Cannot sort table by {column}")
+            if sort:
+                order_by += ", "
```

Comment:
`order_by` should be a list that you later convert to a string using `order_by.join(", ")`.